### PR TITLE
[Oracle] Add global filter in dashboard for Oracle

### PR DESCRIPTION
**Pull request by milan-elastic**
## Proposed commit message
Add a global filter to the dashboard and remove the subfilters on the dataset and a global filter on the dataset is introduced. Filters on data_stream.dataset are applied with a phrase type filter on the dataset that is being used in that particular dashboard.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates https://github.com/elastic/integrations/issues/9323

## Screenshot

<details>
  <summary>Memory Metrics </summary>

Before:

Memory Metrics - before

After:

Memory Metrics - after

</summary>
</details>

<details>
  <summary>Sysmetric </summary>

Before:

Sysmetric - before

After:

Sysmetric - after

</summary>
</details>

<details>
  <summary>System Statistics Metrics </summary>

Before:

System Statistics Metrics - before

After:

System Statistics Metrics - after

</summary>
</details>

<details>
  <summary>Overview </summary>

Before:

Overview - before

After:

Overview - after

</summary>
</details>

<details>
  <summary>Performance Metrics</summary>

Before:

Performance Metrics - before

After:

Performance Metrics - after

</summary>
</details>

<details>
  <summary>Tablespace</summary>

Before:

Tablespace - before

After:

Tablespace - after

</summary>
</details>

### CODE REVIEW

**Comment by agithomas** [comment]
The PR looks good to me.
Please do the needful such that we can merge the changes soon.

File: packages/oracle/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.24.4"
```
[the remark this answers is not in the document]
Comment (milan-elastic) [reply packages/oracle/changelog.yml:2]: > Why not `1.25.0`?

Agree, It should be minor version increment!

**Review by harnish-elastic — APPROVED** [review]
LGTM! 🚀

**Comment by agithomas** [comment]
Since there are multiple datasets in the filter condition, do you want to consider using a Custom Dataset Label ?


image


@tommyers-elastic , what do you think?

**Review by agithomas — APPROVED** [review]
LGTM!

**Pull request merged**